utils/create_dummy_models.py

```diff
@@ -491,6 +524,12 @@ def build_model(model_arch, tiny_config, output_dir):
     if os.path.isdir(processor_output_dir):
         shutil.copytree(processor_output_dir, checkpoint_dir, dirs_exist_ok=True)
 
+    tiny_config = copy.deepcopy(tiny_config)
+
+    if any([model_arch.__name__.endswith(x) for x in ["ForCausalLM", "LMHeadModel"]]):
```

Comment:
copied from `tests/pipelines/test_pipelines_common.py` on `main` - necessary to avoid some test failures